autoPyTorch/optimizer/smbo.py

```diff
@@ -63,27 +70,33 @@ def get_smac_object(
         (SMAC4AC): sequential model algorithm configuration object
 
     """
-    intensifier = Hyperband
+    if initial_budget == max_budget:
+        intensifier = Intensifier
+        intensifier_kwargs: Dict[str, Any] = {'deterministic': True, }
 
-    rh2EPM = RunHistory2EPM4LogCost
-    return SMAC4AC(
+    else:
+        intensifier = Hyperband
+        intensifier_kwargs = {'initial_budget': initial_budget, 'max_budget': max_budget,
+                              'eta': 3, 'min_chall': 1, 'instance_order': 'shuffle_once'}
+    rh2EPM = RunHistory2EPM4LogScaledCost
```

Comment:
To make it consistent with SMAC4HPO. But I am not quite sure, open for discussion :)